src/main/java/httpserver/resourcemanagement/ResourceContentType.java

```diff
@@ -0,0 +1,17 @@
+package httpserver.resourcemanagement;
+
+public enum ResourceContentType {
+
+    JPEG (".jpeg", "image/jpeg"),
+    PNG (".png", "image/png"),
+    GIF (".gif", "image/gif"),
+    TEXT (".txt", "text/plain");
+
+    public final String extension;
+    public final String contentType;
+
+    ResourceContentType(String extension, String contentType) {
+        this.extension = extension;
+        this.contentType = contentType;
+    }
```

Comment:
Could it be a good thing to put final for the parameter? Seems more explicit

http://stackoverflow.com/questions/500508/why-should-i-use-the-keyword-final-on-a-method-parameter-in-java